package.json

```diff
@@ -16,7 +16,7 @@
   "devDependencies": {
     "@edx/brand": "npm:@edx/brand-edx.org@2.0.7",
     "@edx/edx-bootstrap": "2.2.3",
-    "@edx/paragon": "19.25.3",
+    "@edx/paragon": "20.11.1",
```

Comment:
Major Upgrade:
- By adding i18n support to the Paragon design system, we are introducing a peer dependency on react-intl@5.25.0 or greater. This may be a breaking change if:
  - Using v1 of @edx/frontend-platform. 
  - Using older version of react-intl than v5.25.0 directly.
  - Does not use react-intl.
  
 Since we are not using `@edx/frontend-platform` or. `react-intl` in this repo, will this cause a breaking change? The above notes from the release notes for v20.0.0 and may only be referring to MFE's.